docs/FAQs/UseASubscriptionInAJsEvent.md

```diff
@@ -0,0 +1,53 @@
+### Question
+
+How do I access the value of a subscription from within a Reagent or JavaScript event (such as on-click)?
+
+### The Wrong Way
+
+You should NOT do this:
+```clj
+[:input {:type "button" 
+         :value "Click me!"
+         :on-click #(let [sub-val  @(subscribe [:something])]   ;;<--- Eeek
+       ....)}]
+```
+
+because that `subscribe`:
+1. might create a memory leak (the subscription might not be "freed")
+2. makes the event handler impure (it grabs a global value)
+
+
+### The Better Way
+
+If your `on-click` function requires access to re-frame data, the first thing you should do is convert it to a re-frame event and dispatch that instead.
```

Comment:
Not necessarily. It depends on the intention.
E.g. it can be a mostly Reagent component that just needs some data. It doesn't want to affect any outside state, it has its own. In that case, creating an separate re-frame event many make no sense.
I would say here that it's required to deref all subs during the render. And if a user needs a sub value in a JS event, they must create the event handler for each new value of the sub. And the solution presented here is just another way to do that for some of the cases.